src/cbrkit/sim/collections.py

```diff
@@ -86,20 +86,30 @@ def __call__(self, x: Sequence[V], y: Sequence[V]) -> float:
                 return 0.0
 
 
+@dataclass(slots=True, frozen=True)
+class SequenceSim[S: Float](StructuredValue[float]):
+    value: float
+    local_similarities: Sequence[S] = field(default_factory=tuple)
```

Comment:
What is contained in `local_similarities`. The dataclass implies that floats are stored, but the backtracking method returns tuples of values (most likely the mapping I assume). Please check this